docker/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:18.04
+
+LABEL MAINTAINER mh@zenner-iot.com
+
+ARG PG_VERSION=11
+ARG PG_UNIT_VERSION=7.1
+ARG PG_CONFIG=/usr/lib/postgresql/$PG_VERSION/bin/pg_config
+
+RUN apt-get update && \
+    apt-get install -y \
+      build-essential \
+      lsb-release \
+      flex \
+      bison \
+      curl && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list && \
+    curl -fsSL https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
+
+RUN apt-get update && \
+    apt-get install -y \
+      postgresql-server-dev-$PG_VERSION && \
+    rm -rf /var/lib/apt/lists/*
+
+WORKDIR /opt/postgresql-unit
+
+RUN curl -fsSL https://github.com/df7cb/postgresql-unit/archive/$PG_UNIT_VERSION.tar.gz | \
```

Comment:
Hi,
why isn't this using the files from the very git repository you want to have this merged into? Pulling a (versioned) tarball from github instead doesn't seem right.
Alternatively, couldn't you just "apt-get install postgresql-$PG_VERSION-unit" instead?
Christoph

Reply:
D'oh! My mistake...
I didn't check the postgresql repos for your unit extension, just the ubuntu repos :(
Through the docker build, you get the flexibility to test different releases of your extension with different distros and psql versions, but the fact, that the extension is available in the psql apt repo ist making this pull request nearly obsolete...
Nevertheless a Docker Hub Image would be nice and is easily possible with this Dockerfile.